tests/integration/test_vulnerability_detector/test_providers/test_os.py

```diff
@@ -205,7 +205,8 @@ def test_providers_os_extended(configuration, metadata, set_wazuh_configuration,
         evm.check_invalid_option_log(provider_short_name)
 
     evm.check_provider_database_update_start_log(provider_name)
-    evm.check_provider_database_update_finish_log(provider_name, vd.VULN_DETECTOR_FULL_DOWNLOAD_TIMEOUT)
+    evm.check_provider_database_update_finish_log(wazuh_log_monitor=None, provider_name=provider_name,
```

Comment:
```suggestion
    evm.check_provider_database_update_finish_log(provider_name=provider_name,
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/2575/commits/73dd596fd88d841d166213eb00ccae04e87dc800